deltatech_generic_partner_restriction/models/invoicing.py

```diff
@@ -0,0 +1,20 @@
+from odoo import api, fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+    _description = "Restrict Journals"
+
+    @api.onchange("journal_id")
+    def _onchange_journal(self):
+        if self.partner_id.name == "Generic":
+            return {"domain": {"journal_id": [("restriction", "=", False), ("type", "in", ("bank", "cash"))]}}
+        else:
+            return {"domain": {"journal_id": [("type", "in", ("bank", "cash"))]}}
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+    _description = ""
```

Comment:
De ce ai pus lini asta?

Reply:
In acea clasa am creat field-ul "restriction" de tip boolean pentru a restrictiona anumite jurnale, field-ul respectiv trebuie sa fie in acelasi view  unde sunt si jurnalele ca domain-ul sa functioneze, asa am gandit eu

Reply:
de ce ai pus description = ""?

Reply:
Obisnuinta